### ISSUE TITLE
Keyboard Navigation: 4 Navigation / Footer / Header

### ISSUE THREAD

**Issue by fgsw** [joomla/joomla.org#93]
#### Keyboard Navigation in [Joomla.org](https://www.joomla.org/)
![kn-joomla org ](https://github.com/joomla/joomla.org/assets/147295380/e9a01fd7-f23d-4ae4-a8c8-bc4ee5ff1b6a)
No Information which Navigation / Footer / Header is used.

#### Helpful is Keyboard Navigation in Joomla Backend:
![kn-backend](https://github.com/joomla/joomla.org/assets/147295380/74298edb-863e-47f7-86e9-ea3d2049e75d)

**Comment by wilsonge** [comment]
We utilise the following script: https://github.com/paypal/skipto so this will need to be fed into upstream there.

**Comment by brianteeman** [comment]
Not sure why you have tagged me in this. The skipto script is working perfectly and will show the differentiation between headers, navigation and footer regions as shown in the joomla admin ui.

One of the great things about skipto is that it will show you errors in your html which is exactly what it is doing here. If you have multiple regions of the same type (eg navigation) then you MUST differentiate them with an aria-label. That is exactly what is done in the joomla admin ui
` <nav class="main-nav-container" aria-label="Main Menu">`
here we can see that the nav is identified with an aria-label and that the skipto lists that as **Navigation: Main Menu**

In summary - nothing wrong with the skipto script - @wilsonge you need to fix your html
